File: many_small.py
def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
    length = len(nums)
    count_arr = []
    sum = 0
    while length > 0:
        first = nums.pop(0)
        for x in range(len(nums)):
            if first > nums[x]:
                sum += 1
        count_arr.append(sum)
        nums.append(first)
        sum = 0
        length -= 1
    return count_arr

File: many_small.js
var smallerNumbersThanCurrent = function(nums) {
    let len = nums.length;
      let countArr = [];
      let sum = 0;
  while (len > 0){
      let first = nums.shift();
      for (let i = 0; i < nums.length; i++){
          if (first > nums[i]){
              sum += 1;
          }
      }
      countArr.push(sum);
      let last = nums.push(first);
      sum =  0;
      len --;
  }
  return countArr;
  };